src/CloudSeis.jl

```diff
@@ -1168,9 +1168,9 @@ linearframeidx(io, idx::NTuple{N,Int}) where {N} = LinearIndices(io)[CartesianIn
 linearframeidx(io, idx::CartesianIndex) = linearframeidx(io, idx.I)
 linearframeidx(io, idx::Int...) = linearframeidx(io, idx)
 
-cartesianframeidx(io, idx::NTuple{N,Int}) where {N} = CartesianIndex(ntuple(idim->io.axis_lstarts[2+idim] + io.axis_lincs[2+idim]*(idx[idim] - 1), N))
-cartesianframeidx(io, idx::CartesianIndex) = cartesianframeidx(io, idx.I)
-cartesianframeidx(io, idx::Int...) = cartesianframeidx(io, idx)
+contextframeidx(io, idx::NTuple{N,Int}) where {N} = CartesianIndex(ntuple(idim->io.axis_lstarts[2+idim] + io.axis_lincs[2+idim]*(idx[idim] - 1), N))
```

Comment:
`CartesianIndex` is in "Base" Julia.  We can do things like pass a `CartesianIndex` to an array, and it will return the correct index.  So the use of it here is in that context rather than the CloudSeis specfic `ContextIndex` or (now renamed to) `LogicalIndex`.